### PR TITLE
Set User-Agent: HTTP header

### PR DESCRIPTION
**Pull request by saper**

### CODE REVIEW

**Comment by am11** [comment]
Thanks. But what is the real significance of adding UA header in node-sass binary download case?

**Comment by saper** [comment]
It is always good to have it; I noticed we don't have one when I was testing proxy settings and #775 change - http-server just shows "undefined". I am sure corporate local mirrors would really have a proper User-Agent set for their logs.

**Comment by am11** [comment]
Thanks. :+1: 

(having a reason for change is good for future reference :wink:)

**Comment by xzyfer** [comment]
I agree this is useful. Scheduled it for 3.0.0.

File: scripts/install.js [hunk 1]
```diff
@@ -25,6 +25,13 @@ function download(url, dest, cb) {
       cb(typeof err.message === 'string' ? err.message : err);
     };
 
+    var package = require('../package.json');
```
Comment (xzyfer) [inline scripts/install.js]: Please move this to the top with the rest of the `require`s

**Comment by xzyfer** [comment]
Can you please rebase this off master? It's conflicting with recent updates.

**Comment by xzyfer** [comment]
Superseded by https://github.com/sass/node-sass/pull/907

**Pull request closed without merging**